### ISSUE TITLE
how to load robust loss parameters

### ISSUE THREAD

**Issue by 1171000410** [jonbarron/robust_loss_pytorch#26]
Hello, thank you very much for your elegant code. I want to use robust loss to train the model in the first stage and load the pre-trained model in the second stage. I am confused about how to save and load the parameters of AdaptiveLossFunction.

**Comment by jonbarron** [comment]
It should be the same as loading/saving the parameters of any pytorch module. I can't really help you more than that, as I don't use pytorch --- all of the pytorch code I've ever written is in this repository.
